packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.tsx

```diff
@@ -11,6 +11,10 @@ const getCellText = (item: any, column: IColumn): string => {
     value = '';
   }
 
+  if (typeof value === 'boolean') {
+    return String(value);
```

Comment:
I'm admittedly micro-bench-marking here, but this is a hot code path. It looks like calling `toString` is faster compared with concatenation and `String(this)` approaches: https://jsperf.com/boolean-tostring/1.

![image](https://user-images.githubusercontent.com/706967/65927302-10f7ca00-e3ae-11e9-99bf-b92a0a3398a6.png)

Given the above, I think we should consider `value.toString()` here.

Reply:
I'll write the unit test in a follow-up PR if you just want to weigh-in with the above

Reply:
Was using String(x) cause of the null check capabilities but there is already a null check before this, so .toString should work. And if it's faster the better :)

Reply:
> Was using String(x) cause of the null check capabilities but there is already a null check before this, so .toString should work. And if it's faster the better :)

You should be well-guarded against `null` and `undefined` because they won't equate to `boolean`:

```js
typeof null
"object"
typeof undefined
"undefined"
```